### PR TITLE
DOC: Release generation updates for ITK 5.0.0

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: Documentation/Maintenance/Release.md [hunk 1]
```diff
@@ -799,54 +783,15 @@ Release Notes Posts
...
-
-  * `make-changelog.sh`
-  * `prep-emails.py`
-  * `send-emails.sh`
-
-First, create the `changes.txt` file by giving a commit range to
-`make-changelog.sh`:
-
-```sh
-   make-changelog.sh $previous_release $release
```
Comment (dzenanz) [inline Documentation/Maintenance/Release.md]: This part sounds useful.

Comment (thewtex) [reply Documentation/Maintenance/Release.md]: We already have a method to generate the changelog (above). These are left over from the VTK migration.

File: Documentation/Maintenance/Release.md [hunk 2]
```diff
@@ -551,26 +551,19 @@ Create a PR to update
 the new version. This conda recipe downloads the wheel binary packages and
 re-packages them as conda packages.
 
+Note: A `post1` wheel corresponds to bumping the *build.number* field in
+*recipe/meta.yml*. `POST` variables in *bld.bat* and *build.sh* are available
+to specify an options wheel post version.
```
Comment (jhlegarreta) [inline Documentation/Maintenance/Release.md]: to specify an options' wheel post version?

Comment (thewtex) [reply Documentation/Maintenance/Release.md]: Good catch -- should be "optional".

Comment (jhlegarreta) [reply Documentation/Maintenance/Release.md]: Done in ea85dfe.

**Pull request merged**